modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/PartitionsEvictManager.java

```diff
@@ -67,6 +85,11 @@
     /** */
     private final Map<Integer, GroupEvictionContext> evictionGroupsMap = new ConcurrentHashMap<>();
 
+    /**
+     * Evicted partitions for printing to log. It works under {@link #mux}.
+     */
+    private final Map<Integer, Set<EvictPartition>> logEvictPartByGrps = new HashMap<>();
```

Comment:
What's the point of introducing new class EvictPartition? Why can't we use Map<partId, reason> ?

Reply:
In the beginning, I thought about compacting storage in memory, but thinking now, I think I won’t win because we are only interested in a couple of fields.

Reply:
I can redo the use of map.